api/utils/ogc_translate.py

```diff
@@ -18,6 +22,37 @@ def set_namespaces(xml_element):
 
     return xml_element
 
+def get_hysds_status_from_wps(wps_status):
+    """
+    Translate WPS job status to HySDS job status
+    :param status: (str) WPS job status
+    :return status: (str) HySDS job status
+    """
+    job_status = None
+    if wps_status not in WPS_STATUSES:
+        statuses = ", ".join(str(status) for status in WPS_STATUSES)
+        raise ValueError("Invalid WPS status: {}. Valid values are: {}".format(wps_status, WPS_STATUSES))
+    elif wps_status == "Accepted":
+        job_status = "job-queued"
+    elif wps_status == "Running":
+        job_status = "job-started"
+    elif wps_status == "Succeeded":
+        job_status = "job-completed"
+    elif wps_status == "Failed":
+        job_status = "job-failed"     
+    elif wps_status == "Dismissed":
+        job_status = "job-revoked"
+    elif wps_status == "Deduped":
+        job_status = "job-deduped"
+    elif wps_status == "Offline":
+        job_status = "job-offline"
+    elif wps_status == "Deleted":
+        job_status = None
+    else:
+        job_status = None
+
+    return job_status
```

Comment:
And with the above change, we can make this function smaller. 

```suggestion
    if wps_status not in WPS_TO_HYSDS_JOB_STATUS_MAP:
        statuses = ", ".join(str(status) for status in WPS_STATUSES)
        raise ValueError("Invalid WPS status: {}. Valid values are: {}".format(wps_status, WPS_STATUSES))
    return WPS_TO_HYSDS_JOB_STATUS_MAP.get(wps_status)
```

Reply:
If the 1-to-1 mapping is confirmed, this is definitely easier.